test/k8sT/Tunnels.go

```diff
@@ -123,8 +128,14 @@ var _ = Describe("K8sValidatedTunnelTest", func() {
 
 		It("Check Geneve mode", func() {
 
+			err := kubectl.CiliumInstall("cilium_ds_geneve.jsonnet")
```

Comment:
Why is the `BeforeEach` removed here? There is no explanation as to why this is being done.

Reply:
The BeforeEach was part of the test. Checking the installation of the daemon set it's part of the test itself, not something that need to be in there to the test to run. 

+ I want to avoid the current issue that the BeforeEach Fail not gather the logs.